mobile/src/bundles/auth/store/slice.ts

```diff
@@ -2,15 +2,20 @@ import { createSlice } from '@reduxjs/toolkit';
 
 import { DataStatus } from '~/bundles/common/enums/enums';
 import { type ValueOf } from '~/bundles/common/types/types';
+import { type UserRole } from '~/bundles/users/enums/enums';
 
 import { signUp } from './actions';
 
 type State = {
     dataStatus: ValueOf<typeof DataStatus>;
+    isSignedIn: boolean;
```

Comment:
we got pretty the same task with store, need to cooperate and make the same structure)